source/scss/clarendon.scss

```diff
@@ -0,0 +1,21 @@
+/*
+ * Clarendon
+ *
+ */
+@import 'shared';
+
+%clarendon,
+.clarendon {
+  @extend %filter-base;
+  filter: contrast(1.2) saturate(1.35);
```

Comment:
I was playing with this one and I think I got it super close!! Try `contrast(1.1) saturate(1.05);` for the filter